mkosi/sandbox/resources/mkosi-sandbox.md

```diff
@@ -0,0 +1,151 @@
+% mkosi-sandbox(1)
+%
+%
+
+# NAME
+
+mkosi-sandbox — Run commands in a custom sandbox
+
+# SYNOPSIS
+
+`mkosi-sandbox [options…] command [arguments]`
+
+# DESCRIPTION
+
+`mkosi-sandbox` runs the given command in a custom sandbox. The sandbox is configured
+by specifying command line options that configure individual parts of the sandbox.
+
+If no command is specified, `mkosi-sandbox` will start `bash` in the sandbox.
+
+Note that this sandbox is not designed to be a security boundary. It's intended purpose
+is to allow running commands in an isolated environment so they are not affected by the
+host system.
+
+# OPTIONS
+
+`--tmpfs`
+:   Takes a single argument. Mounts a new tmpfs in the sandbox at the given path.
```

Comment:
Not our usual style, but makes the sentences on arity unnecessary and therefore the whole man page if not shorter at least less repetitive and also drives home that `--arg=val` is not supported, since in our usualy style all CLI args are given as `--arg=`
```suggestion
`--tmpfs target`
:   Mounts a new tmpfs in the sandbox at the given path.
```